src/js/components/awardv2/visualizations/amounts/AggregatedAwardAmountsInfo.jsx

```diff
@@ -60,16 +60,16 @@ export default class AggregatedAwardAmountsInfo extends React.Component {
                 <div className="award-amounts__data">
                     <span>Awards that Reference this IDV</span><span>{awardAmounts.idvCount + awardAmounts.contractCount}</span>
                 </div>
-                <a
-                    href="/"
-                    className="award-viz__link">
-                    <div className="award-viz__link-icon award-viz__link-icon_hidden">
+                <button
+                    onClick={() => this.props.jumpToSection('referenced-awards')}
+                    className="award-viz__button">
```

Comment:
Another convo might be worth having around the best practice for passing `fns` to `onClick`. This is typically what I do as well when passing a `fn` with a certain parameter, but [the docs](https://reactjs.org/docs/faq-functions.html) indicate there are performance implications to doing this.

Reply:
Good call, removed arrow functions from render